packages/rich-text/src/plugins/CommandPalette/components/CommandPrompt.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+
+import { BLOCKS } from '@contentful/rich-text-types';
+import { PlateRenderLeafProps } from '@udecode/plate-core';
+import { RichTextEditor } from 'types';
+
+import { isNodeTypeSelected } from '../../../helpers/editor';
+import { trimLeadingSlash } from '../utils/trimLeadingSlash';
+import { CommandList } from './CommandList';
+
+export const CommandPrompt = (props: PlateRenderLeafProps) => {
+  const query = trimLeadingSlash(props.text.text);
```

Comment:
We need to use `React.useMemo` on `query`, otherwise the referential inequality is going to make `CommandList` re-render on every render of `CommandPrompt`, when it might not need to.